cylc/flow/job_runner_handlers/documentation.py

```diff
@@ -0,0 +1,412 @@
+# THIS FILE IS PART OF THE CYLC WORKFLOW ENGINE.
+# Copyright (C) NIWA & British Crown (Met Office) & Contributors.
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with this program.  If not, see <http://www.gnu.org/licenses/>.
+
+"""
+This file is used to auto-generate some reference documentation for the
+job runner plugin interface.
+
+Note the class contained here is just for documentation purposes and is
+not intended to be subclassed.
+"""
+
+import re
+from typing import (
+    Iterable,
+    List,
+    Tuple,
+)
+
+
+class ExampleHandler():
+    """Documentation for writing job runner handlers.
+
+    Cylc can submit jobs to a number of different job runners
+    (aka batch systems) e.g. Spice and PBS. For a list of built-in integrations
```

Comment:
```suggestion
    (aka batch systems) e.g. Slurm and PBS. For a list of built-in integrations
```
("Spice"?)

Reply:
Ah jeez, good spot.

(Spice is the name of an internal system which uses Slurm)